pkg/controller/serviceaccount/ca_configmap_controller.go

```diff
@@ -0,0 +1,203 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package serviceaccount
```

Comment:
I think I would prefer if this was in a new directory in pkg/controller/certificates

Reply:
directory pkg/controller/certificates already exists, what about pkg/controller/sa-certificates or maybe pkg/controller/serviceaccount/certificates?